transcom/mymove
# INT B-20496 Safety Move Authentication Error Message

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20496)

## Summary

This branch updates the error message that displays when an office user attempts to access a safety move when they do not have the adequate permissions to view safety moves.

The new page specifies that the page is not accessible, instead of the generic "Oops!" message that pops up for all other page errors.

### How to test

1. Create an admin user at adminlocal:3000
2. Access DBeaver or psql, find your admin user in the `admin_users` table, and check the `super` column, this gives the user the ability to grant safety move permissions to office users.
3. In the admin dashboard, click "Office Users" and create a new user with your desired roles, be sure to include "Service Counselor".
4. After creating, find the user in the "Office Users" list and click "edit", you should now see a "Safety Move" checkbox, check it and save the changes.
5. Login to officelocal as the SC user, click the "Customer Search" tab, search for any string, then click the "Add Customer" button that pops up.
![Screenshot 2024-11-20 at 12 36 29 PM](https://github.com/user-attachments/assets/e081daf9-402e-4a4d-b0c1-2ca1bca41410)
6. Create the customer, ensuring that the "Is this a safety move?" checkbox at the very top is checked.
7. Create the orders as needed, then you should be directed to the move details page for your new customer/move. Copy the URL.
8. Login as another user without safety move privileges and navigate to the URL, you should now see the updated error screen



## Screenshots
![Screenshot 2024-11-20 at 12 28 12 PM](https://github.com/user-attachments/assets/dfe34d4b-6338-44c9-acd6-9029735a4551)
